src/integrations/job-runners/README.md

```diff
@@ -0,0 +1,70 @@
+# Job Runners
+
+## Intro
+
+Spoke has several functions that need to be run in the background after
+an API request has returned. Strategies for executing this background
+work depend on the deployment environment. This job runner API allows
+organizations to customize background jobs to their deployment without
+introducing conflicts with MoveOn/main.
+
+
+## Tasks vs Jobs
+
+"Tasks" are lightweight fire-and-forget operations, while "Jobs" are
+long-running operations that are tracked in the JobRequest table.
+
+While most queueing systems should be able to handle both types of background
+processing, we are keeping them separate for now to allow for different execution
+strategies. For example, it might be reasonable to always run tasks in-process
+outside of AWS Lambda while still pushing Jobs to a separate worker process.
+
+Note: This api may change in the future.
```

Comment:
Maybe worth adding something about a task's maximum payload should be. Lambda's is, I believe, 256K so maybe 255K for wrapping space.